pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -71,6 +71,20 @@ type Plan struct {
 	Phases []Phase `json:"phases"`
 }
 
+// ParameterValueType specifies the type of a parameter value.
+type ParameterValueType string
+
+const (
+	// StringValueType is used for parameter values that are provided as a string.
+	StringValueType ParameterValueType = "string"
+
+	// ListValueType is used for parameter values that described a list of values.
+	ListValueType ParameterValueType = "list"
```

Comment:
these are externalized to the user and are used in yaml (which is json).  Is there a reason we would chose values which are not defined by json schema?   at least where we can.    I would expect that "array" 

I would also expect that map is more understood than dict...

Reply:
I second what Ken said. These names seem natural to me, but would probably be surprising to someone without Python background.

Reply:
I tried to use verbs that are closer to YAML. But the [YAML spec uses "collections" and "sequences"](https://yaml.org/spec/1.2/spec.html#id2759963), [JSON uses "collections" and "lists"](https://www.json.org/json-en.html) . Good point regarding JSON schema, I'll use the names that are used there.